scipy/stats/tests/test_distributions.py

```diff
@@ -206,6 +206,37 @@ def test_vonmises_expect():
     assert np.issubdtype(res.dtype, np.complexfloating)
 
 
+@pytest.mark.parametrize('loc', [-np.pi, 0, np.pi])
+@pytest.mark.parametrize('kappa', [1, 10, 100, 1000])
+def test_vonmises_fit_all(kappa, loc):
```

Comment:
How long do these tests take? If above 1-2 seconds, we should mark with slow.

Reply:
0.5 - 1 second on my machine.